src/Persistence/MassTransit.EntityFrameworkCoreIntegration/Saga/ContainerSagaDbContextFactory.cs

```diff
@@ -0,0 +1,34 @@
+namespace MassTransit.EntityFrameworkCoreIntegration.Saga
+{
+    using MassTransit.Saga;
+    using Microsoft.EntityFrameworkCore;
+
+
+    public class ContainerSagaDbContextFactory<TContext, TSaga> :
```

Comment:
This is nice, so the container handles the lifetime of the dbcontext. So would this mention of consume lifetime scoped in the docs be obsolete now?

https://masstransit-project.com/usage/sagas/persistence.html#consume-lifetime-scoped-dbcontext

Reply:
Yes, the documentation will need to be updated entirely once the syntax is completed. But yes, letting the container manage the lifecycle is a key aspect, so dependencies can put DbContext on their constructor and get it injected.